en/docs/Administer/ProductAdministration/DataSources/WorkingWithDatabases/SettingUp/admin-setting-up-postgresql.md

```diff
@@ -10,12 +10,12 @@ The following sections describe how to set up PostgreSQL to replace the default
 Follow the steps below to set up a PostgreSQL database.
 
 1.  Install PostgreSQL on your computer as follows:
-    ![](attachments/126562544/126562571.png)
+    ![](/assets/attachments/126562544/126562571.png)
```

Comment:
Folder 126562544 too is missing. Will need to check all these images individually.

Reply:
This image is still missing.